src/main/java/mate/academy/spring/service/RoleService.java

```diff
@@ -0,0 +1,10 @@
+package mate.academy.spring.service;
+
+import java.util.Optional;
+import mate.academy.spring.model.Role;
+
+public interface RoleService {
+    void add(Role role);
```

Comment:
```suggestion
    Role add(Role role);
```
it is better to return Role